app/presenters/admin_access_presenter.rb

```diff
@@ -0,0 +1,18 @@
+class AdminAccessPresenter < SimpleDelegator
```

Comment:
This `Presenter` isn't currently pulling its weight yet, but hopefully with view + edit it will. If it doesn't, then well, I'll just stick these few methods back in the controller.